README.md

```diff
@@ -14,76 +14,54 @@
   </a>
 </p>
 
-Computational Pathology Toolbox developed by TIA Lab
+Computational Pathology Toolbox developed at the TIA Centre
 
-Getting Started
-===============
+## All Users
 
-First, install
-- OpenSlide [here](https://openslide.org/download/).
-- OpenJPEG [here](https://www.openjpeg.org/), using
-`conda install -c conda-forge openjpeg>=2.3.0`.
+This package is for those interested in digital pathology: including graduate students, medical staff, members of the Tia Centre and of Pathlake, and anyone, anywhere, who may find it useful. We will continue to improve this package, taking account of developments in pathology, microscopy, computing and related disciplines. Send comments and criticisms to **[tia@dcs.warwick.ac.uk](mailto:tialab@dcs.warwick.ac.uk)**.
```

Comment:
Please capitalise TIA, --> members of "TIA" Centre

Reply:
Done